src/views/ChooseFactionView/ChooseFactionList.ts

```diff
@@ -0,0 +1,106 @@
+import BRETTONIA from '../../assets/factions/fantasy/brettonia.jpg';
+import CATHAY from '../../assets/factions/fantasy/cathay.jpg';
+import CHAOS from '../../assets/factions/fantasy/chaos.jpg';
+import DARK_ELVES from '../../assets/factions/fantasy/dark_elves.jpg';
+import DWARF from '../../assets/factions/fantasy/dwarf.jpg';
+import EMPIRE from '../../assets/factions/fantasy/empire.jpg';
+import HIGH_ELVES from '../../assets/factions/fantasy/high_elves.jpg';
+import ORCS_AND_GOBLINS from '../../assets/factions/fantasy/orcs_and_goblins.jpg';
+import NIPPON from '../../assets/factions/fantasy/nippon.jpg';
+import SKAVEN from '../../assets/factions/fantasy/skaven.jpg';
+import ARABS from '../../assets/factions/dark_ages/arab.jpg';
+import CELTS from '../../assets/factions/dark_ages/celt.jpg';
+import NORMANS from '../../assets/factions/dark_ages/normans.jpg';
+import ROMANS from '../../assets/factions/dark_ages/romans.jpg';
+import SAXONS from '../../assets/factions/dark_ages/saxons.jpg';
+import VIKINGS from '../../assets/factions/dark_ages/vikings.jpg';
+
+const FantasyFactions = [
+  {
+    id: 'Brettonia',
+    name: 'Brettonia',
+    img: BRETTONIA,
+  },
+  {
+    id: 'Cathay',
+    name: 'Cathay',
+    img: CATHAY,
+  },
+  {
+    id: 'Chaos',
+    name: 'Chaos',
+    img: CHAOS,
+  },
+  {
+    id: 'Dark_Elves',
+    name: 'Dark Elves',
+    img: DARK_ELVES,
+  },
+  {
+    id: 'Dwarf',
+    name: 'Dwarf',
+    img: DWARF,
+  },
+  {
+    id: 'Empire',
+    name: 'Empire',
+    img: EMPIRE,
+  },
+  {
+    id: 'High_Elves',
+    name: 'High Elves',
+    img: HIGH_ELVES,
+  },
+  {
+    id: 'Orcs_and_Goblins',
+    name: 'Orcs and Goblins',
+    img: ORCS_AND_GOBLINS,
+  },
+  {
+    id: 'Nippon',
+    name: 'Nippon',
+    img: NIPPON,
+  },
+  {
+    id: 'Skaven',
+    name: 'Skaven',
+    img: SKAVEN,
+  },
+];
+const DarkAgesFactions = [
+  {
+    id: 'Arabs',
+    name: 'Arabs',
+    img: ARABS,
+  },
+  {
+    id: 'Celts',
+    name: 'Celts',
+    img: CELTS,
+  },
+  {
+    id: 'Normans',
+    name: 'Normans',
+    img: NORMANS,
+  },
+  {
+    id: 'Romans',
+    name: 'Romans',
+    img: ROMANS,
+  },
+  {
+    id: 'Saxons',
+    name: 'Saxons',
+    img: SAXONS,
+  },
+  {
+    id: 'Vikings',
+    name: 'Vikings',
+    img: VIKINGS,
+  },
+];
+
+export default {
+  FantasyFactions,
+  DarkAgesFactions,
```

Comment:
Minor thing that will matter later, we should have these keys correspond to the IDS for [fantasy](https://github.com/Tremors-Of-War/tremors-of-war/blob/4e60ae9e9af87ee56006b0bd77ef1ed078401aca/src/data.json#L673-L674) and [dark ages](https://github.com/Tremors-Of-War/tremors-of-war/blob/4e60ae9e9af87ee56006b0bd77ef1ed078401aca/src/data.json#L2747) in our database.

I realise those IDs are not in our existing naming scheme, but maybe we should change that in our `data.json`